lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -139,7 +139,16 @@ def store_column(column_title, format_string, new_data):
             ),
         )
 
-    store_column("minimum_length (-)", "%d", minimum_length)
+    if not (None in minimum_length):
```

Comment:
More idiomatic placement of `not`?
```suggestion
    if None not in minimum_length:
```

Reply:
Yeah, that's better.